dev/dll/SharedHelpers.cpp

```diff
@@ -284,21 +279,14 @@ bool SharedHelpers::IsInFrameworkPackage()
     }();
 
     return isInFrameworkPackage;
-#else
-    return false;
-#endif
 }
 
 // Platform scale helpers
 winrt::Rect SharedHelpers::ConvertDipsToPhysical(winrt::UIElement const& xamlRootReference, const winrt::Rect& dipsRect)
 {
     try
     {
-#if defined(BUILD_WINDOWS)
-        const auto scaleFactor = static_cast<float>(xamlRootReference.XamlRoot().RasterizationScale());
-#else
```

Comment:
Is RasterizationScale not in the public XamlRoot?

Reply:
It was added after 19H1 snapped.

Reply:
It shows it as available here: https://docs.microsoft.com/en-us/uwp/api/Windows.UI.Xaml.XamlRoot.RasterizationScale. I would leave the code and make it runtime conditional.

Reply:
ok. For 19h1 and above i'll use this api.